script/people/migrate_ai_an_to_attested_status.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# This script identifies and updates the 'American Indian Status' verification type
+# fields in People, moving their state from 'outstanding', 'review', 'pending', 'unverified',
+# and 'rejected' to 'attested'.
+#
+# The script operates in two modes:
+# 1. `report`: Generates a report of people with 'American Indian Status' verification type fields in the aforementioned states.
+# 2. `data_fix`: Updates the 'American Indian Status' verification types.
+#
+# @example Run in report mode to identify people with negative American Indian Status states
+#   MODE="report" bundle exec rails runner script/people/migrate_ai_an_to_attested_status.rb
+#
+# @example Run in data_fix mode to update the American Indian Status verification types to attested
+#   MODE="data_fix" bundle exec rails runner script/people/migrate_ai_an_to_attested_status.rb
+
+mode = ENV['MODE']
+
+# Validate mode and provide usage guidance
+unless mode
+  puts <<~USAGE
+    ************************************************************
+    Please provide a valid argument. Must be one of: report, data_fix
+    To generate a report of People with negative American Indian Status states, run:
+      MODE='report' bundle exec rails runner script/migrate_ai_an_to_attested_status.rb
+    To update the American Indian Status verification types to attested, run:
+      MODE='data_fix' bundle exec rails runner script/migrate_ai_an_to_attested_status.rb
+    ************************************************************
+  USAGE
+  exit
+end
+
+# Start timing the process
+start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
```

Comment:
FYI, Sai K recently added a nice model for us to use when benchmarking performance: https://github.com/ideacrew/enroll/blob/trunk/app/models/caches/benchmark_cache.rb

You can see an example of its use here:
https://github.com/ideacrew/enroll/blob/44c1bfab5438ff40b8af022920c41f7cf2d948b9/script/tax_household_groups/copy_application_id_to_application_hbx_id.rb#L10